nucypher-core-python/nucypher_core/__init__.pyi

```diff
@@ -431,6 +434,40 @@ class ThresholdDecryptionRequest:
         ...
 
 
+class E2EEThresholdDecryptionRequest:
+
+    def decryption_request(self) -> ThresholdDecryptionRequest:
+        ...
+
+    def response_encrypting_key(self) -> PublicKey:
+        ...
+
+    @staticmethod
+    def from_bytes(data: bytes) -> E2EEThresholdDecryptionRequest:
+        ...
+
+    def __bytes__(self) -> bytes:
+        ...
+
+
+class EncryptedThresholdDecryptionRequest:
+    def id(self) -> int:
```

Comment:
This is also a property. Also perhaps better named `ritual_id` to match Rust API.

Reply:
✔️